src/current/v23.2/live-migration-service.md

```diff
@@ -670,6 +750,116 @@ These steps assume you have already followed the overall steps to [prepare for m
 1. Any writes that were made during the cutover will have been missed on CockroachDB. Use [MOLT Verify]({% link {{ page.version.version }}/molt-verify.md %}) to identify the inconsistencies. These will need to be manually reconciled.
 {% endcomment %}
 
+### Monitor cutover
+
+You can monitor your cutover attempts with the following commands.
+
+### `connections list`
```

Comment:
Did you mean to make these H4s or leave them at H3?

Reply:
🤦 Good point. I'll fix in a follow-up PR.